src/Voxel.MiddyNet.HttpJsonBodyParserMiddleware/Constants.cs

```diff
@@ -0,0 +1,7 @@
+namespace Voxel.MiddyNet.HttpJsonBodyParserMiddleware
+{
+    public class Constants
+    {
+        public const string BodyContextKey = "Body";
```

Comment:
I would put this directly in the main middleware class, as we´ve done with the TraceContext.